.CI/build_all

```diff
@@ -10,6 +10,7 @@ if [ -z "$GH_TOKEN" ]; then
     python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
+    conda config --add channels r
```

Comment:
If that is acceptable we should add that to the templates.

@msarahan is there a plan to move the packages in the `r` channel to the default channel?